docs/HISTORY.txt

```diff
@@ -4,7 +4,9 @@ Changelog
 1.3.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add title to mobilebutton when description is not present.
+  For accessibility reasons, title attribute on element mustn't be empty.
```

Comment:
@bierik Small fix in sentence:  For accessibility reasons, title attribute on element `can't` be empty.